frontend/sass/_actions.scss

```diff
@@ -1,3 +1,7 @@
+a {
+  text-decoration-skip: ink;
```

Comment:
snuck in this minor styling change. It makes the underline skip descenders, like on the `g` here:

<img width="72" alt="screen shot 2017-08-24 at 12 27 14 pm" src="https://user-images.githubusercontent.com/697848/29679581-a0a2b366-88c7-11e7-8691-865e85798d64.png">